src/test/java/org/assertj/core/internal/strings/strings_assertMixedCase_Test.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2021 the original author or authors.
+ */
+package org.assertj.core.internal.strings;
+
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.assertj.core.error.ShouldBeLowerCase.shouldBeLowerCase;
+import static org.assertj.core.test.TestData.someInfo;
+import static org.assertj.core.util.FailureMessages.actualIsNull;
+
+import org.assertj.core.internal.StringsBaseTest;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Tests for <code>{@link org.assertj.core.internal.Strings#assertLowerCase(org.assertj.core.api.AssertionInfo, CharSequence)} </code>.
```

Comment:
Favor imports for elements in `@link`, e.g.:

```suggestion
 * Tests for <code>{@link Strings#assertMixedCase(AssertionInfo, CharSequence)}</code>.
```

Also please add:

```java
@DisplayName("Strings assertMixedCase")
```

like mentioned in the [contributing section](https://github.com/assertj/assertj-core/blob/main/CONTRIBUTING.md).

I would also favor the GIVEN/WHEN/THEN structure and parameterized tests for successful and failure cases to reduce tests verbosity.